viewer/packages/pointclouds/src/PointCloudIntersection.ts

```diff
@@ -29,6 +29,7 @@ export type PointCloudIntersection = {
   distanceToCamera: number;
   /**
    * Annotation Id of the intersected object within a pointcloud. (0 if not applicable)
+   * @deprecated Use `volumeRef` instead
```

Comment:
```suggestion
   * @deprecated Use `volumeMetadata` instead
```

Reply:
Weird, I can't see this on my end :thinking: